pontoon/base/templates/project_selector.html

```diff
@@ -13,33 +13,38 @@
     </div>
     {% if not project %}
     <div class="sort clearfix">
-      <span class="asc">Name<i class="fa"></i></span>
-      <span>Website<i class="fa"></i></span>
-      <span>Progress<i class="fa"></i></span>
+      <span class="name asc">Name<i class="fa"></i></span>
+      <span class="latest">Latest activity<i class="fa"></i></span>
+      <span class="progress">Progress<i class="fa"></i></span>
     </div>
     {% endif %}
     <ul>
       {% for p in projects %}
       <li class="clearfix{% if project and project == p %} current{% endif %}">
-        {% if url_prefix %}
-        <a href="/{{ url_prefix }}/{{ p.slug }}/" class="clearfix">
+        <span class="name"
+          data-slug="{{ p.slug }}"
+          {% if p.details %}data-details="{{ p.details }}"{% endif %}>
+          {% if url_prefix %}<a href="/{{ url_prefix }}/{{ p.slug }}/" class="clearfix">{% endif %}
+            {{ p.name }}
+          {% if url_prefix %}</a>{% endif %}
+        </span>
+        {% if not project %}
+        <span class="latest">
+          {% set latest_activity = p.latest_activity(locale) if locale else p.latest_activity() %}
+          {% include 'latest_activity.html' %}
```

Comment:
This pattern for reusing template code isn't a great idea because the included template and this template share a template context, leading to possible variable name collisions, as well as making it very difficult to figure out where a variable is being set (is it set in the template? In the view?).

You should replace this with a [macro](http://jinja.pocoo.org/docs/dev/templates/#macros).